google-cloud-tasks/Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
+
```

Comment:
Probably want to add this as well:

```ruby
if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.1")
  # WORKAROUND: builds are failing on Ruby 2.0.
  # We think this is because of a bug in Bundler 1.6.
  # Specify a viable version to allow the build to succeed.
  gem "jwt", "~> 1.5"
end
```